tests/test_data_query.py

```diff
@@ -28,7 +28,8 @@
 
 from rcsbapi.search import rcsb_attributes as attrs
 from rcsbapi.data import DataSchema, DataQuery
-from rcsbapi.config import ApiSettings
+from rcsbapi.config import config
+from rcsbapi.const import const
 
```

Comment:
```suggestion
# Suppress autocomplete warning
config.SUPPRESS_AUTOCOMPLETE_WARNING = True
```
I think suppressing the warnings for test files would be good so the test logs are cleaner or changing the paths to full paths to get rid of the warnings

Reply:
Hmm, I actually kind of like the fact that they are included in the CI/CD testing logs, since that gives us a history we can look back on to see if and how anything may have changed. In general, I've found it valuable to make the testing logs as extensive as possible, for debugging purposes.